console/src/telemetry/Actions.ts

```diff
@@ -145,18 +191,31 @@ const loadConsoleOpts = () => {
 
     return dispatch(requestAction(url, options) as any).then(
       (data: Telemetry[]) => {
-        if (data.length !== 0) {
+        if (data.length) {
+          const { hasura_uuid, console_state } = data[0];
+
           dispatch({
             type: SET_HASURA_UUID,
-            data: data[0].hasura_uuid,
+            data: hasura_uuid,
           });
-          globals.hasuraUUID = data[0].hasura_uuid;
+          globals.hasuraUUID = hasura_uuid;
+
           dispatch({
             type: SET_CONSOLE_OPTS,
-            data: data[0].console_state,
+            data: console_state,
           });
-          globals.telemetryNotificationShown = !!data[0].console_state
-            ?.telemetryNotificationShown;
+          globals.telemetryNotificationShown = !!console_state?.telemetryNotificationShown;
+
+          if (!console_state?.console_notifications) {
+            dispatch({
+              type: UPDATE_CONSOLE_NOTIFICATIONS,
```

Comment:
Don't understand why we're doing it?

Reply:
when we don't have the `console_notification` defined on the DB. this just initializes it on redux.